lib/service_provider.dart

```diff
@@ -62,11 +68,39 @@ class ServiceProvider {
             localDataRepository: getIt.get(),
           ));
 
+    GetIt.instance.registerFactory<MainPageCubit>(
```

Comment:
Could you please use `getIt` variable which is declared on the beginning of the method ?

Reply:
Done